web/styles/search.css

```diff
@@ -2,144 +2,219 @@
     width: 100%;
     height: var(--header-height);
 
+    .navbar-search {
+        margin: 5.5px 0;
+        border-radius: 5px;
+        position: absolute;
```

Comment:
It's a little hard to mess with this in the Firefox inspector, but elsewhere in this file, `.navbar-search` is floated--which won't have any effect if it's actually an absolutely-positioned element.

Reply:
Yup! Good catch. Removed the float.